CONTRIBUTING.md

```diff
@@ -0,0 +1,182 @@
+# Contributing Guidelines
+
+This documentation contains a set of guidelines to help you during the contributing process. We are happy to welcome all the contributors who's willing to enhance/add features to this project. Feel free to contribute and remember, no contribution is too small.
+
+## Submitting Contributions :woman_technologist: :man_technologist:
+
+### Step 1 : Find an issue
+
+- Take a look at the Existing Issues or [create your own Issues!](#creating-an-issue)
+- Wait for the Issue to be assigned to you after which you can start working on it.
+- Note : Every change in this project should/must have an associated issue.
+
+### Step 2: Fork the repository
+
+- Fork this Repository. This will create a local copy of this repository on your Github Profile.
+
+![fork](https://user-images.githubusercontent.com/66238394/114071871-3cbb1080-98bf-11eb-94aa-4b5f811ece77.png)
+
+### Step 3: Clone the repository:
+
+- Clone this Github repo. Open up the GitBash/Command Line and type in:
```

Comment:
`GitHub`. And replace `repo` with `repository`. Use `terminal` instead of Git Bash and Command Line